mocks.json

```diff
@@ -1,6 +1,6 @@
 {
   "base_directory": "github.com/Scalingo/go-scalingo",
-  "base_package": "github.com/Scalingo/go-scalingo/v5",
+  "base_package": "github.com/Scalingo/go-scalingo/v6",
```

Comment:
I'm surprised this file changes but not `mocks_sig.json`. Are you sure you didn't forgot to commit it?